Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/SubmissionRepository.kt

```diff
@@ -121,17 +123,20 @@ class SubmissionRepository @Inject constructor(
     suspend fun asyncRegisterDeviceViaTAN(tan: String) {
         val registrationData = submissionService.asyncRegisterDeviceViaTAN(tan)
         LocalData.registrationToken(registrationData.registrationToken)
-        updateTestResult(registrationData.testResult)
         LocalData.devicePairingSuccessfulTimestamp(timeStamper.nowUTC.millis)
+        updateTestResult(registrationData.testResult)
```

Comment:
Why was the call order changed?

Reply:
Not necessary anymore. The implementation is independent from the order.